### ISSUE TITLE
Broken OAuth in fixed/broke notifications for Concourse >= 6.0

### ISSUE THREAD

**Issue by failytaler** [arbourd/concourse-slack-alert-resource#61]
It seems like Auth mechanism changed a bit for new versions and resource has an error when trying to authorize.
```
2020/05/27 08:56:32 error getting last build status: error connecting to Concourse: oauth2: cannot fetch token: 404 Not Found
Response: 404 page not found
```

**Comment by arbourd** [comment]
That sucks! Sorry to hear that. I'm not using Concourse right now so it's a bit hard to pindown. Are you willing to take a look?

**Comment by failytaler** [comment]
Actually, I don't think I'm capable)
The issue is that in new skymarshal package they removed %/sky/token path.
https://github.com/concourse/concourse/blob/master/skymarshal/skyserver/skyserver.go

**Comment by ldbl** [comment]
Hi guys I just found few routes are completely removed from version 6.
https://github.com/concourse/concourse/blob/324c3af716013b13eb9780d108810d2816960139/skymarshal/skyserver/skyserver.go#L46
Compared to version 6 
https://github.com/concourse/concourse/blob/v6.1.0/skymarshal/skyserver/skyserver.go
```
handler.HandleFunc("/sky/userinfo", server.UserInfo)
handler.HandleFunc("/sky/token", server.Token)```
Those 2 are removed . 
I tried to search for a token and I found it here. 
https://github.com/concourse/concourse/blob/a4c9e8ed8205218b1fdd8646f5a107a6fbbb670e/cmd/concourse/web.go#L106

**Comment by arbourd** [comment]
I just pushed what might be a quick fix:
https://hub.docker.com/layers/arbourd/concourse-slack-alert-resource/fix-6.0/images/sha256-17799fc3daf3126d50008baaf5f82216a6bc11e33f82c10f66a6c4b69e65122a

Can you try tag `arbourd/concourse-slack-alert-resource:fix-6.0` and let me know how it goes? I'll try to get a cluster up and working sometime this week, but I can't at the very minute.

I'm hoping the only thing that changed is the url and not the guts 🤞

**Comment by failytaler** [comment]
Nope, an error is the same. Will try to debug it as well soon.

**Comment by arbourd** [comment]
It looks like they added a CSRF token. The URL changed, but I'm getting 401s now instead of 404s 😱

**Comment by arbourd** [comment]
Fix is in `v0.11.0`. I tested it on a local 6.1, but if there any other auth issues please let me know.

**Comment by smoke** [comment]
Everything works great, thank you!
